python/tests/res/job_queue/test_job_dispatch.py

```diff
@@ -0,0 +1,96 @@
+from tests import ResTest
+from res.enkf import ResConfig
+from ecl.util.test import TestAreaContext
+
+import os, stat
+import json
+from subprocess import Popen
+import time
+import psutil
+
+class JobDispatchTest(ResTest):
+
+    def setUp(self):
+        self.case_directory = self.createTestPath("local/poly_normal/")
+
+    def testTerminateJobs(self):
+
+    
+
+        with TestAreaContext("job_dispatch_test") as work_area:
+            work_area.copy_directory(self.case_directory)
+
+            #executes it self recursively and sleeps for 100 seconds
+            with open("dummy_executable", "w") as f:
+                f.write("#!/usr/bin/env python\n"\
+                        "import sys, os, time\n"\
+                        "counter = eval(sys.argv[1])\n"\
+                        "if counter > 0:\n"\
+                        "    os.fork()\n"\
+                        "    os.execv(sys.argv[0],[sys.argv[0], str(counter - 1) ])\n"\
+                        "else:\n"\
+                        "    time.sleep(100)\n"
+                    )
```

Comment:
not important, but in this situations it could be easier to use the triple quotes `"""`